application/src/test/resources/org/opentripplanner/apis/gtfs/expectations/realtime-trip.json

```diff
@@ -48,13 +44,13 @@
         "scheduledDeparture": 41400
       },
       "datedOrigin": {
-        "realtimeDeparture": 0
+        "realtimeDeparture": 41400
       },
       "timetabledDestination": {
         "scheduledArrival": 42000
       },
       "datedDestination": {
-        "realtimeArrival": 900
+        "realtimeArrival": 42000
       },
```

Comment:
This changes also seems a bit random.

Reply:
This change is certainly wrong. I am testing the data should be replaced on a REPLACEMENT trip here.